komodo/cli.py

```diff
@@ -60,14 +60,15 @@ def _print_timings(
 
 def _main(args):
     timings: List[Tuple[str, datetime.timedelta]] = []
-
     abs_prefix = Path(args.prefix).resolve()
 
     data = Data(extra_data_dirs=args.extra_data_dirs)
 
     if args.download or (not args.build and not args.install):
         start_time = datetime.datetime.now()
-        git_hashes = fetch(args.pkgs, args.repo, outdir=args.downloads, pip=args.pip)
+        git_hashes = fetch(
+            args.pkgs.content, args.repo.content, outdir=args.downloads, pip=args.pip
+        )
```

Comment:
Where does this change stem from? Is it a newer black, or an editor that has done something with the whitespace?